src/routes/safe/components/Settings/Appearance/index.tsx

```diff
@@ -21,6 +21,10 @@ const Container = styled(Block)`
   padding: ${lg};
 `
 
+const StyledPrefixedEthHashInfo = styled(PrefixedEthHashInfo)`
+  margin-bottom: 1em;
```

Comment:
Why are you adding this?

Reply:
I needed to remove the a wrapper `Paragraph` in the Appearance page because of a warning that was showing in the console. I thought of keeping the previous paragraph's bottom margin though.

Reply:
Why don't you just remove the `Paragraph`?

Reply:
I did. And it removed the `PrefixedEthHashInfo` margin bottom. This style was intended to restore the removed margin.

Reply:
Ah, I follow you. I thought you were modifying the wrapper directly.